Error configuring libedit

I am getting a configure error in `spack install libedit` with the following entry in the log:
```
checking for tgetent in -lncurses... no
checking for tgetent in -lcurses... no
checking for tgetent in -ltermcap... no
configure: error: libncurses, libcurses, or libtermcap is required!
```
This may be a result of the recent change to provide libncursesw, also discussed in #3727.

--- a/var/spack/repos/builtin/packages/libedit/package.py
+++ b/var/spack/repos/builtin/packages/libedit/package.py
@@ -28,8 +28,9 @@ from spack import *
 class Libedit(AutotoolsPackage):
     """An autotools compatible port of the NetBSD editline library"""
     homepage = "http://thrysoee.dk/editline/"
-    url      = "http://thrysoee.dk/editline/libedit-20150325-3.1.tar.gz"
+    url      = "http://thrysoee.dk/editline/libedit-20170329-3.1.tar.gz"
 
+    version('3.1-20170329', 'c57a0690e62ef523c083598730272cfd')
     version('3.1-20160903', '0467d27684c453a351fbcefebbcb16a3')
     version('3.1-20150325', '43cdb5df3061d78b5e9d59109871b4f6')
 
@@ -38,3 +39,6 @@ class Libedit(AutotoolsPackage):
     def url_for_version(self, version):
         url = "http://thrysoee.dk/editline/libedit-{0}-{1}.tar.gz"
         return url.format(version[-1], version.up_to(-1))
+
+    def configure_args(self):
+        return ['LIBS=-lncursesw']

